client/src/index.js

```diff
@@ -0,0 +1,44 @@
+//Load libraries
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import reportWebVitals from './reportWebVitals';
+//Load css
+import './index.css';
+//Load React components
+import App from './App';
+import NotFound from './components/NotFound';
+import RegistrationContainer from './components/RegistrationContainer';
+import ForgotPasswordContainer from './components/ForgotPasswordContainer';
+import DashboardPage from './components/DashboardPage';
+import AddEntryPage from './components/AddEntryPage';
+
+/*
+  The Main Routing file for this app  
+
+  The "*" path will match only when no other routes do. (Redirect to 404 page not found)
+  When routes are nested the UI should be nested too. Meaning the parent UI will still display
+*/
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <Routes>
+          <Route path="/" element={<App />} > 
+  
+          </Route>
```

Comment:
cosmetic thing: why is this not self-closed as the routes down below `/>` instead of `</Route>`

Reply:
I was experimenting with sub-routes. I left it like this in case someone else wants to add sub-routes in the future.